### PR TITLE
Makes puppeteer's minions non-dense (kill bodyblock)

### PR DESCRIPTION
**Pull request by Barnet2**
## About The Pull Request
What it says on the tin.
## Why It's Good For The Game
This makes the caste extremely toxic to fight as you essentially have 4 xenos bodyblocking your escape. To achieve this now you actually need your team.
## Changelog
:cl:
balance: Made puppeteer's minions non-dense
/:cl:

### CODE REVIEW

**Comment by hivemuscle** [comment]
https://github.com/tgstation/TerraGov-Marine-Corps/pull/15916#issue-2308989298

This PR nerfs puppeteer damage. Author claims that it will make puppeteer a "deadly wall". Question: how can you pass through a wall? Bring the damage back then.

**Comment by jonash07** [comment]
honey wake up, the peak is here

**Comment by Wisemonster** [comment]
> [#15916 (comment)](https://github.com/tgstation/TerraGov-Marine-Corps/pull/15916#issue-2308989298)
> 
> This PR nerfs puppeteer damage. Author claims that it will make puppeteer a "deadly wall" rather then "insta-death". Question: how can you pass through a wall? Bring the damage back then.

I meant this more along the lines of "bullet wall", not "Literally can't walk through it" wall. Puppeteers puppets don't need to do 1 less damage compared to spiderlings to be useful.

**Comment by SimplySmiley123** [comment]
Finally something to make the tier 2 not a straight upgrade to the tier 3 counterpart.

**Comment by Lefunny** [comment]
Barnet trying to kill a caste.mp3, it's damage already got nerfed puppeteer don't need another nerf

**Comment by iforgotmeword** [comment]
This is going to make puppets so toxic to fight ;-; (without aiming)

**Comment by hivemuscle** [comment]
> > [#15916 (comment)](https://github.com/tgstation/TerraGov-Marine-Corps/pull/15916#issue-2308989298)
> > This PR nerfs puppeteer damage. Author claims that it will make puppeteer a "deadly wall" rather then "insta-death". Question: how can you pass through a wall? Bring the damage back then.
> 
> I meant this more along the lines of "bullet wall", not "Literally can't walk through it" wall. Puppeteers puppets don't need to do 1 less damage compared to spiderlings to be useful.

Puppets have 200 hp and no bullet armor. How can they be a "bullet wall"? Buff their armor then. Also, this kind of a "wall" is useless to xenos, because it runs slower than them. Also, one puppet costs 125 biomass which you can't just regenerate. Also, you must lead the puppets to frontline by yourself, risking to die because you are frail. Also, resin applied to puppeter applies not to puppets.

In other words, stop fantasizing. Puppeteer is not a support caste to provide some sort of a "wall". It kills lone marines inside the maze. It must kill them efficiently. It must PUNISH them for being too cocky, chasing low hp xenos while ignoring the entire hive because because meraderm vali. In other aspects this caste is weak (for example, puppeteer is risking very much while taking down marines in the open field, hive already got a hunter for this). Puppets can't even fracture a marine and a chest fracture is worse than a death. So why would you even choose puppeteer and not hunter after this stupid nerfs?

**Comment by Woker2004** [comment]
> In other words, stop fantasizing. Puppeteer is not a support caste to provide some sort of a "wall". It kills lone marines inside the maze. It must kill them efficiently. It must PUNISH them for being too cocky, chasing low hp xenos while ignoring the entire hive because meraderm vali. In other aspects this caste is weak (for example, puppeteer is risking very much while taking down marines in the open field, hive already got a hunter for this). Puppets can't even fracture a marine and a chest fracture is worse than a death. So why would you even choose puppeteer and not hunter after these stupid nerfs?

Puppeteer is a T2 that frankly, should of never been merged, it is a widow but it doesn't worry about TFoot, you can get puppets quicker than widow can frankly recover, it has better individual minions than widow, and did I mention it can stun easier than widow? The problem is it does what widow can do but as a T2, and frankly honest, is better than widow which is a T3

**Comment by hivemuscle** [comment]
> > In other words, stop fantasizing. Puppeteer is not a support caste to provide some sort of a "wall". It kills lone marines inside the maze. It must kill them efficiently. It must PUNISH them for being too cocky, chasing low hp xenos while ignoring the entire hive because meraderm vali. In other aspects this caste is weak (for example, puppeteer is risking very much while taking down marines in the open field, hive already got a hunter for this). Puppets can't even fracture a marine and a chest fracture is worse than a death. So why would you even choose puppeteer and not hunter after these stupid nerfs?
> 
> Puppeteer is a T2 that frankly, should of never been merged, it is a widow but it doesn't worry about TFoot, you can get puppets quicker than widow can frankly recover, it has better individual minions than widow, and did I mention it can stun easier than widow? The problem is it does what widow can do but as a T2, and frankly honest, is better than widow which is a T3

Don't talk about what should or should not have been merged, marines got aoe shattering plasma gun after all. You are hallucinating (or spreading disinformation on purpose, idk), saying that puppeteer can get puppets quicker than widow. At least because "Stitch Puppet" ability got 25 seconds cooldown, compared to the 15 second cooldown of the Widow's similar minion spawn ability. And plasma is much cheaper to get than biomass. If you know not - you can no more refurbish roundstart bodies, only DNR marines. Also, widow got "Leash Ball" ability that plays a massive role in holding marine inside the maze. Gorillion times better than puppeteer's slowdown ability, which is not so strong and costs you a valuable biomass. Also, widow now can build (TMed), so cope PR.

**Review by Lumipharon — CHANGES_REQUESTED** [review]
Don't they just stack all on 1 tile together if they're nondense? So you can't even see how many there are?

**Comment by Looking4bros** [comment]
I've found that trailblazer grenades had fixed much of my issues playing vali against puppeters. Really the only thing that can save me from them. Just need the right tool for the job methinks.
